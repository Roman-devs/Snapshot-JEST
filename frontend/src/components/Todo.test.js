import React from "react";
import renderer from 'react-test-renderer';
import Board from "./Board";
import "jest-styled-components"

it('should render correctly', () => {
    const totoTest = [
        {
            id: "12345",
            description: "asodh"
        },
        {
            id: "49102=",
            description: "kjdkdjkd"
        }]

    const kuul = renderer.create(<Board todos={totoTest}
                                        title={"TestKram"}
                                        onDelete={() => console.log("Test1")}
                                        onAdvance={console.log("Test2")} />)
    expect(kuul).toMatchSnapshot();
});