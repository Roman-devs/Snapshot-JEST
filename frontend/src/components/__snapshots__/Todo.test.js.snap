// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<section
  className="Board__Wrapper-sc-1yu0kxh-0 gmKzFA"
>
  <h2>
    TestKram
  </h2>
  <ul>
    <li>
      <section
        className="Todo__Wrapper-sc-1u29sm1-0 hwKVuW"
      >
        <span
          className="Todo__Description-sc-1u29sm1-1 bkfpda"
        >
          asodh
        </span>
        <button
          className="sc-bdfBwQ dxYAvz"
          onClick={[Function]}
        >
          Delete
        </button>
      </section>
    </li>
    <li>
      <section
        className="Todo__Wrapper-sc-1u29sm1-0 hwKVuW"
      >
        <span
          className="Todo__Description-sc-1u29sm1-1 bkfpda"
        >
          kjdkdjkd
        </span>
        <button
          className="sc-bdfBwQ dxYAvz"
          onClick={[Function]}
        >
          Delete
        </button>
      </section>
    </li>
  </ul>
</section>
`;
